mlrun/utils/helpers.py

```diff
@@ -1410,6 +1411,26 @@ def is_running_in_jupyter_notebook() -> bool:
     return ipy and "Terminal" not in str(type(ipy))
 
 
+def create_ipython_display():
+    """
+    Create an IPython display object and fill it with initial content.
+    We can later use the returned display_id with the update_display method to update the content.
+    If IPython is not installed, a warning will be logged and None will be returned.
+    """
+    if is_ipython:
+        import IPython
+
+        display_id = uuid.uuid4().hex
+        content = IPython.display.HTML(
+            f'<div id="{display_id}">Temporary Display Content</div>'
+        )
+        IPython.display.display(content, display_id=display_id)
+        return display_id
+
+    # returning None if IPython is not installed
+    logger.warn("IPython is not installed, cannot create IPython display")
```

Comment:
I think this log is not relevant if `is_ipython` is `False` then we are not using Jupyter and this logs an unrelated warning